documentation/resources/firewalld_helpers.md

```diff
@@ -0,0 +1,21 @@
+# firewalld_helpers
+
+## Actions
+
+- `:update`
+
+## Properties
+
+| Name        | Type                               | Default | Description  |
+| ------      | ------                             | ------- | ------------ |
+| version     | String                             |         |              |
```

Comment:
Would like to see descriptions here as this is going to be the primary place someone is going to look for understanding of what each property is for and how to use them.

Reply:
I will update all the docs accordingly, but is there a way of generating this?  All the information that needs to be in the docs is already available to the resource, so it should be possible somehow, right?

Reply:
I'm not aware of the tool but agree it would be nice. @ramereth @Xorima @damacus are you guys aware of a tool that'll help generate the docs for resources?

Reply:
I hacked [this](https://github.com/bmhughes/chef_doc_builder) together and it does work but it's not exactly what you'd call well documented.

Reply:
Docs are updated now. @bmhughes: thanks for the tool.